emission/analysis/modelling/tour_model/similarity.py

```diff
@@ -41,10 +41,10 @@ def __init__(self, data, radius, old=True):
                 # print "a is %s" % a
                 t = a
                 try:
-                    start_lon = t.start_loc["coordinates"][0]
-                    start_lat = t.start_loc["coordinates"][1]
-                    end_lon = t.end_loc["coordinates"][0]
-                    end_lat = t.end_loc["coordinates"][1]
+                    start_lon = t.start_place.location["coordinates"][0]

```

Comment:
I added some logging statements, and it is pretty clear that the one in representatives is the one being called. Which makes sense because t.start_place is an object ID, so I would expect t.start_place.location to fail.

How did you test this to confirm that it works?